step/step.go

```diff
@@ -19,7 +19,7 @@ const (
 	key = `{{ .Arch }}-cocoapods-cache-{{ checksum "**/Podfile.lock" }}`
 
 	// Cached path
-	path = "Pods"
+	path = "**/Pods"
```

Comment:
Will we need to make an equivalent change in the restore-cocoapods-cache step?

Reply:
Not really, the restore step is only concerned with the cache key. It restores the file structure found in the archive, so once the save step creates an archive including the nested Pods folder, the restore step will simply restore that.